### DISCUSSION TITLE
Another plus for sane-airscan

### DISCUSSION THREAD

**Discussion by debiantriage** [alexpevzner/sane-airscan#296] [General]
Greeting @alexpevzner and all airscan users,

On a user's laptop I prevent network printers being exposed by having `deny-interfaces=<laptop_wireless_interface>` in `/etc/avahi/avahi-daemon.conf`. This is because she become confused by seeing a long list of printers, the list being mostly due to my various investigations. A printer for her sole use is made available on the lo interface. Via a script, specific network printers can be accessed and specific jobs (single-sided, duplex, booklet etc)  sent.

Having crippled Avahi how does scanning go on? Have a guess!
```
brian@patspc:~$ scanimage -L
device `net:192.168.7.40:epson2:libusb:003:015' is a Epson RX420 flatbed scanner
device `airscan:w0:HP ENVY 4500 series' is a WSD HP ENVY 4500 series ip=192.168.7.235
```
Yes, you guessed correctly; sane-airscan falls back to using WSD and offering resolutions 75,100,200, 300, 600 and 1200. Splendid.

**Comment by alexpevzner** [comment]
Hi @debiantriage,

> Yes, you guessed correctly; sane-airscan falls back to using WSD 

:))

Sorry, didn't notice it before (strange, GitHub sends e-mail when somebody creates an Issue, but not Discussion).

> This is because she become confused by seeing a long list of printers

It's a real problem of zeroconf. Users of big corporative networks are being confusing by seeing a long list of discovered devices.

I wonder, how MacOS addresses this problem? (MacOS is important, because zeroconf is driven by Apple's effort)
